x/wasm/internal/keeper/staking_test.go

```diff
@@ -404,6 +406,106 @@ func TestReinvest(t *testing.T) {
 	assertSupply(t, ctx, keeper, contractAddr, "200000", sdk.NewInt64Coin("stake", 236000))
 }
 
+func TestQueryStakingInfo(t *testing.T) {
+	// STEP 1: take a lot of setup from TestReinvest so we have non-zero info
+	initInfo := initializeStaking(t)
+	defer initInfo.cleanup()
+	ctx, valAddr, contractAddr := initInfo.ctx, initInfo.valAddr, initInfo.contractAddr
+	keeper, stakingKeeper, accKeeper := initInfo.wasmKeeper, initInfo.stakingKeeper, initInfo.accKeeper
+	distKeeper := initInfo.distKeeper
+
+	// initial checks of bonding state
+	val, found := stakingKeeper.GetValidator(ctx, valAddr)
+	require.True(t, found)
+	//initPower := val.GetDelegatorShares()
+	assert.Equal(t, val.Tokens, sdk.NewInt(1000000), "%s", val.Tokens)
```

Comment:
nit: this and others: for better failure messages define expected value first:  `Equal(t, expectedVal, gotVal)